integration_tests/src/main/python/hybrid_parquet_test.py

```diff
@@ -290,6 +305,458 @@ def test_hybrid_parquet_filter_pushdown_timestamp(spark_tmp_path):
+    "(floor(double1) == 1)",
+    # Double, Int:
+    # Round,ArrayRepeat
+    "(round(double1) == 3)",
+    # Double, Double:
+    # NaNvl,Pow
+    "(nanvl(double1, 1) == 1)",
+    "(pow(double1, double2) < 1.0)",
+    # Double, Long:
+    # Round
+    "(round(double1, 1) == 3.1)",
+    # Date:
+    # LastDay, WeekOfYear,DayOfMonth, DayOfWeek, DayOfYear, Month, Quarter, Second, UnixMicros, UnixMillis, UnixSeconds, WeekDay, Year
+    "(last_day(date1) == date1)",
+    "(weekofyear(date1) == 1)",
+    "(dayofmonth(date1) == 1)",
+    "(dayofweek(date1) == 1)",
+    "(dayofyear(date1) == 1)",
+    "(month(date1) == 1)",
+    # Date, Int
+    # AddMonths
+    "(add_months(date1, 1) == date3)",
+    # Date, Date:
+    # DateDiff
+    "(date_diff(date1, date2) == 1)",
+    # unit, Int, Date:
+    # DateAdd,DateSub
+    "(date_add(date1, 1) == date2)",
+    "(date_sub(date1, 1) == date2)",
+    # Array[Int], lambda:
+    # ArrayExists,ArrayForAll
+    "(exists(array_int_1, x -> x == 1))",
+    "(forall(array_int_1, x -> x == 1))",
+    # Array[Int], Int:
+    # ArrayContains,ArrayPosition,ArrayRemove,ElementAt
+    "(array_contains(array_int_1, 1))",
+    "(array_position(array_int_1, 1) == 1)",
+    "(array_remove(array_int_1, 1) == array_int_1)",
+    "(element_at(array_int_1, 1) == 1)",
+    # Array[Int]:
+    # ArrayDistinct,ArrayMax,ArrayMin,ArraySort,Size
+    "(array_distinct(array_int_1) == array_int_1)",
+    "(array_max(array_int_1) == 1)",
+    "(array_min(array_int_1) == 1)",
+    "(sort_array(array_int_1) == array_int_1)",
+    "(size(array_int_1) == 1)",
+    # Array[Int], Array[Int]:
+    # ArrayExcept,ArrayIntersect,ArraysZip
+    "(array_except(array_int_1, array_int_2) == array_int_1)",
+    "(array_intersect(array_int_1, array_int_2) == array_int_1)",
+    "(arrays_zip(array_int_1, array_int_2) == " + 
+    "array(struct(1 as array_int_1, 1 as array_int_2), struct(2 as array_int_1, 2 as array_int_2)))",
+    # Array[Array[Int]]:
+    # Flatten
+    "(flatten(array_array_int_1) == array(1,2,1,2))",
+    # Array[Int], Array[Int], lambda:
+    # ZipWith
+    "(zip_with(array_int_1, array_int_2, (x, y) -> x + y) == array(2,4))",
+    # String:
+    # Ascii,BitLength,Hex,Length,Lower,Reverse,SoundEx,StringTrim,StringTrimLeft,StringTrimRight,Upper
+    "(ascii(str1) == 97)",
+    "(bit_length(str1) == 16)",
+    "(hex(str1) == '616263')", # abc -> 616263
+    "(length(str1) == 2)",
+    "(lower(str1) == 'abc')",
+    "(reverse(str1) == 'cba')",
+    "(soundex(str1) == 'A120')", # abc -> A120
+    "(trim(str1) == 'abc')",
+    "(ltrim(str1) == 'abc')",
+    "(rtrim(str1) == 'abc')",
+    "(upper(str1) == 'ABC')",
+    # Int, Int:
+    # ArrayRepeat,BitwiseAnd,BitwiseOr,BitwiseXor,EqualNullSafe,EqualTo,GreaterThan,GreaterThanOrEqual,
+    # Greatest,Least,LessThan,LessThanOrEqual,Remainder,ShiftLeft,ShiftRight
+    pytest.param("(array_repeat(1, 2) == array_int_1)", marks=pytest.mark.xfail(reason='VeloxRuntimeError')),
```

Comment:
Seems a bug in `hybrid-execution`, banned this expression, will file a issue later.

Reply:
It's not issue about `array_repeat`. And filed https://github.com/NVIDIA/spark-rapids/issues/12504